dot/rpc/helpers.go

```diff
@@ -44,3 +49,47 @@ func LocalRequestOnly(r *rpc.RequestInfo, i interface{}) error {
 	}
 	return errors.New("external HTTP request refused")
 }
+
+func SnakeCaseFormat(method string) (string, error) {
+	parts := strings.Split(method, ".")
+	if len(parts) < 2 {
+		return "", fmt.Errorf("invalid rpc method format %s, should be 'module.FunctionName'", method)
+	}
+
+	service, funcName := parts[0], parts[1]
+	funcName = strings.ToLower(string(funcName[0])) + funcName[1:]
+	return strings.Join([]string{service, funcName}, "_"), nil
+}
+
+func RPCValidator(cfg *HTTPServerConfig, validate *validator.Validate) func(r *rpc.RequestInfo, i interface{}) error {
```

Comment:
This function doesn't need to be public.
`rpcValidator`

Reply:
Done!